dashboard-app/src/features/dashboard/components/Title.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import styled from 'styled-components';
+import { H3 } from '../../../lib/ui/components/Headings';
+import { SpacingEnum, ColoursEnum } from '../../../lib/ui/design_system';
+
+export type TitleString = [string, string];
+
+type Props = {
+  title: TitleString
+};
+
+const TitleContainer = styled.div`
+  display: flex;
+  justify-content: space-between;
+  padding: ${SpacingEnum.small};
+  background-color: ${ColoursEnum.white};
+`;
+
+const Text = styled(H3)`
+  text-align: left;
+  margin-bottom: 0;
+`;
+
+export const Title: React.FunctionComponent<Props> = (props) => {
```

Comment:
What is the difference, semantically, between a title and a heading? Genuine question.

The dashboard pages each have one main "heading", and one main "sub-heading" (I'm using "heading" but only because I don't know what better word to use).

Just wondering whether `Title` is actually an accurate name for what this is, because it doesn't feel like it.

Reply:
Yeah, `Title` is a terrible name. Open to suggestions :)

Reply:
Off the top of my head, `Subheading`, `VisualisationHeading` (🤢), `TabHeading` (probs would have to move it if we call it that). Of the three I think the latter most closely matches how we're currently using it, but 🤷🏻‍♀️

Reply:
The same component is being used in `StackedBarChart`, so the latter won’t work. I’ll go for the one that doesn’t make you feel sick.